platform/android/MapboxGLAndroidSDK/.gitignore

```diff
@@ -0,0 +1,2 @@
+lint-baseline.xml
+lint/lint-baseline-local.xml
```

Comment:
@zugaldia they should, but in practice baselines include absolute paths (pointing to the files containing an error) which are different in local and in CI, so we need to keep both 😞